tests/manual-test-cases/Group19-DRS-Disabled/19-3-DRS-Disabled-Placement.robot

```diff
@@ -55,15 +54,20 @@ Teardown VCH And Cleanup Nimbus
 
 Simple Placement
     Install VIC Appliance To Test Server
+    Log To Console  Deploy VIC to the VC cluster
+    Set Environment Variable  GOVC_URL  ${vc1-ip}
+    Set Environment Variable  TEST_URL_ARRAY  ${vc1-ip}
+    Set Environment Variable  TEST_RESOURCE  cls3
+    Set Environment Variable  TEST_TIMEOUT  30m
+
     ${vch_host}=  Get VM Host Name  %{VCH-NAME}
 
     ${stressed_hosts}=  Create List  ${vch_host}
-    :FOR  ${i}  IN RANGE  2
-    \   ${ip}=  Deploy Stress Container And Return Host IP  stresser${i}
-    \   Should Not Contain  ${stressed_hosts}  ${ip}
-    \   Append To List  ${stressed_hosts}  ${ip}
+    ${ip}=  Deploy Stress Container And Return Host IP  stresser
+    Should Not Contain  ${stressed_hosts}  ${ip}
+    Append To List  ${stressed_hosts}  ${ip}
 
-    # 1 VCH host + 2 stressed hosts out of 4 total, leaving one
+    # 1 VCH host + 1 stressed hosts out of 3 hosts total, leaving one
     # clean host to which a new container should relocate
     ${len}=  Get Length  ${stressed_hosts}
     Should Be Equal As Integers  ${len}  3
```

Comment:
`Should Be Equal As Integers  ${len}  2`

Reply:
Also quick comment for line 78: s/VC/VCH

Reply:
Good catches, thanks